### PR TITLE
Update changelog for v3.6.1

### PR DESCRIPTION
**Pull request by lfbrock**
(Wait until the final v3.6.1 is cut before merging)

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -2,9 +2,14 @@
...
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released bi-monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
-## Release v3.6.0  
+## Release v3.6.1  
 
-Release date: 2017-01-16
+### Notes on Patch Release
+
+ - **v3.6.1, released 2017-01-17**
+   - Fixed a performance regression when sending many notifications at once, upgrading only necessary if you notice a large delay when sending `@all` or `@channel` notifications in channels with a lot of users
```
Comment (jasonblais) [inline source/administration/changelog.md]: I thought the second part, `upgrading only necessary...`, was still referring to the fix itself.

Maybe we could break it into two bullet points so it's more obvious 3.6.1 upgrade is only necessary for the case you described:

```
 - **v3.6.1, released 2017-01-17**
   - Fixed a performance regression when sending many notifications at once
   - Note: Upgrading to Mattermost v3.6.1 only necessary if you notice a large delay when sending `@all` or `@channel` notifications in channels with a lot of users
```

**Comment by esethna** [comment]
+1, after the final date of release is updated

File: source/administration/changelog.md [hunk 2]
```diff
@@ -2,9 +2,17 @@
...
 
-## Release v3.6.0  
+## Release v3.6.1  
 
-Release date: 2017-01-16
+### Notes on Patch Release
+
+ - **v3.6.1, released 2017-01-17**
+   - Fixed a performance regression when sending many notifications at once (for example, when `@all` or `@channel` is used in a channel with many users)
+   - Fixed an issue where the config flag for the CLI was not backwards compatible
```
Comment (esethna) [inline source/administration/changelog.md]: Should "config" be formatted? Propose `--config`

File: source/administration/changelog.md [hunk 3]
```diff
@@ -2,9 +2,17 @@
...
+   - Fixed an upgrade issue for MySQL databases using certain character sets
```
Comment (esethna) [inline source/administration/changelog.md]: Do we want to say which character sets?

File: source/administration/changelog.md [hunk 4]
```diff
@@ -2,9 +2,17 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released bi-monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
-## Release v3.6.0  
+## Release v3.6.1  
 
-Release date: 2017-01-16
+### Notes on Patch Release
+
+ - **v3.6.1, released 2017-01-17**
```
Comment (esethna) [inline source/administration/changelog.md]: date needs to be updated

File: source/administration/changelog.md (hunk 2 above)
Comment (lfbrock) [inline source/administration/changelog.md]: There's different alternatives for the config flag, including `--config` (new) and `-config` (old). I think it's appropriate to just reference it as the "config flag", unless we want to reword the whole sentence.

**Pull request merged**